### PR TITLE
Add VZ 1.6.11 CLI download instructions

### PR DESCRIPTION
**Pull request by pfmackin**
Add VZ 1.6.11 CLI download instructions since it is needed to generate Kubernetes YAML manifests from cluster resources related to OAM.

### CODE REVIEW

File: doc/experimental/oam-to-kubernetes.md [hunk 1]
```diff
@@ -1,11 +1,32 @@
 # OAM to Kubernetes Mappings
 
-### Version: v0.0.1-draft
+### Version: v0.0.2-draft
+This document explains how to generate Kuberenetes YAML files from OAM-related resources that are running in a Kubernetes cluster.
 
-## Verrazzano CLI export command
+## Download Verrazzano 1.6.11 CLI
 Verrazzano provides a CLI command that you can use to facilitate the migration of an OAM application to be managed as a collection of Kubernetes objects.
+If you don't have the 1.6.11 CLI (or later), you need to download it to export the OAM resources to Kubernetes manfests.
```
Comment (mgianatagh) [inline doc/experimental/oam-to-kubernetes.md:8]: "export the Kubernetes objects that were generated for an OAM application"

File: doc/experimental/oam-to-kubernetes.md [hunk 2]
```diff
@@ -1,11 +1,32 @@
...
 
-The command vz export oam will output the YAML for each Kubernetes object that was generated as a result of deploying an OAM application. The generated YAML is sanitized so that it can be used to deploy the application. Fields such as creationTimestamp, resourceVersion, uid, and status are not included in the output.
+Download the CLI for Linux-AMD as follows:
+```
+curl -LO https://github.com/verrazzano/verrazzano/releases/download/v1.6.11/verrazzano-1.6.11-linux-amd64.tar.gz
```
Comment (mgianatagh) [inline doc/experimental/oam-to-kubernetes.md:12]: Should we make this command generic to any linux environment, or just use amd64 as an example?

Comment (mgianatagh) [reply doc/experimental/oam-to-kubernetes.md:12]: I just checked the VZ doc, and it just has amd64, so probably ok as is.

File: doc/experimental/oam-to-kubernetes.md [hunk 3]
```diff
@@ -1,11 +1,32 @@
...
+### Version: v0.0.2-draft
+This document explains how to generate Kuberenetes YAML files from OAM-related resources that are running in a Kubernetes cluster.
 
-## Verrazzano CLI export command
+## Download Verrazzano 1.6.11 CLI
 Verrazzano provides a CLI command that you can use to facilitate the migration of an OAM application to be managed as a collection of Kubernetes objects.
+If you don't have the 1.6.11 CLI (or later), you need to download it to export the OAM resources to Kubernetes manfests.
 
-The command vz export oam will output the YAML for each Kubernetes object that was generated as a result of deploying an OAM application. The generated YAML is sanitized so that it can be used to deploy the application. Fields such as creationTimestamp, resourceVersion, uid, and status are not included in the output.
+Download the CLI for Linux-AMD as follows:
```
[the remark this answers is not in the document]
Comment (pfmackin) [reply doc/experimental/oam-to-kubernetes.md:10]: fixed

**Review by mgianatagh — APPROVED** [review]

**Review by George-Aeillo — APPROVED** [review]

**Pull request merged**